master/introduction/index.md

```diff
@@ -21,8 +21,29 @@ you can achieve fine-grained control over communications
 between containers, virtual machine workloads, and 
 bare metal host endpoints.
 
-Proven in production at scale, {{site.prodname}} features 
-integrations with Kubernetes, OpenShift, Docker, 
-Mesos, DC/OS, and OpenStack.
+Proven in production at scale, {{site.prodname}} v3.0 features 
+integrations with Kubernetes, OpenShift, and OpenStack.
 
-<a href="/{{page.version}}/getting-started/" class="btn btn-primary btn-lg">Get started</a>
\ No newline at end of file
+> **Note**: For integrations with the  Mesos, DC/OS, and Docker 
+> orchestrators, use [Calico v2.6](/v2.6/introduction/). We plan 
+> to resume support for these orchestrators in a future 
+> v3.x release.
+{: .alert .alert-info}
+
+# Get started
+
+<a href="/{{page.version}}/getting-started/kubernetes/" class="btn btn-primary btn-sm">Kubernetes</a>
+<a href="/{{page.version}}/getting-started/openshift/installation" class="btn btn-primary btn-sm">OpenShift</a>
+<a href="/{{page.version}}/getting-started/openstack/" class="btn btn-primary btn-sm">OpenStack</a>
+
+# How it works
+
+![Calico overview diagram]({{site.baseurl}}/images/calico-arch-gen.svg){: width="60%" }
```

Comment:
I suggest adding a brief high-level overview of how Calico functions to explain the diagram before defining each component. I think something like this would tie it together a bit more:

------------------------------------------------

Calico leverages the routing and iptables firewall capabilities native to the Linux kernel. The calico/node agent runs on each host and programs the kernel with the desired routes and network policy rules for each workload. All traffic to or from individual containers, VMs, and hosts traverses these in-kernel rules before being routed to its destination.

Workload, policy, and configuration information is stored in a central key/value store and is written automatically by one of several orchestrator specific integrations. Configuration and policy can also be modified explicitly using the calicoctl command line tool.